process.sh

```diff
@@ -15,7 +15,7 @@ main() {
     INPUT="$FILES/input"
     OUTPUT="$FILES/output"
 
-    for DATASET in cytokit linnarsson dries wang cao spraggins; do
+    for DATASET in cytokit linnarsson dries wang cao spraggins annotations_spleen_0510; do
```

Comment:
```suggestion
    for DATASET in cytokit linnarsson dries wang cao spraggins satija; do
```
Just a convention, but we've been generally naming by PI.